src/libs/ajax/WorkspaceDataService.ts

```diff
@@ -22,5 +22,11 @@ export const WorkspaceData = signal => ({
     const res = await fetchWDS(`${instanceId}/tsv/v0.2/${recordType}`, _.merge(authOpts(), { signal }))
     const blob = await res.blob()
     return blob
+  },
+  uploadTsv: async (instanceId, recordType, file) => {
+    const formData = new FormData()
+    formData.set('records', file)
+    const res = await fetchWDS(`${instanceId}/tsv/v0.2/${recordType}`, _.mergeAll([authOpts(), { body: formData, signal, method: 'POST' }]))
+    return res
```

Comment:
For more consistency between data fetching functions, is it possible to return the JSON response here? So that consumers don't see the Response object?